python-dependencies.yaml

```diff
@@ -0,0 +1,89 @@
+name: python-dependencies
+buildsystem: simple
+build-commands:
+- pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "scipy" "matplotlib" "pyxdg" "evdev" "pyudev"
+sources:
+- type: file
+  url: https://files.pythonhosted.org/packages/1c/41/91955188e97c7b85fbaac6bbf4e33de028899e0aa31bdce99b1afe2eeb17/pycairo-1.26.0.tar.gz
+  sha256: 2dddd0a874fbddb21e14acd9b955881ee1dc6e63b9c549a192d613a907f9cbeb
+- type: file
+  url: https://files.pythonhosted.org/packages/b0/b4/bc2baae3970c282fae6c2cb8e0f179923dceb7eaffb0e76170628f9af97b/wheel-0.42.0.tar.gz
+  sha256: c45be39f7882c9d34243236f2d63cbd58039e360f85d0913425fbd7ceea617a8
+- type: file
+  url: https://files.pythonhosted.org/packages/c7/c3/55076fc728723ef927521abaa1955213d094933dc36d4a2008d5101e1af5/wheel-0.42.0-py3-none-any.whl
+  sha256: 177f9c9b0d45c47873b619f5b650346d632cdc35fb5e4d25058e09c9e581433d
+- type: file
+  url: https://files.pythonhosted.org/packages/5a/62/007b63f916aca1d27f5fede933fda3315d931ff9b2c28b9c2cf388cd8edb/numpy-1.26.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+  sha256: f25e2811a9c932e43943a2615e65fc487a0b6b49218899e62e426e7f0a57eeda
+- type: file
+  url: https://files.pythonhosted.org/packages/6b/d4/d62ce38ba00dc67d7ec4ec5cc19d36958d8ed70e63778715ad626bcbc796/scipy-1.11.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
How will this work for aarch64? You are building on both arches right now.

Reply:
so also apply to numpy above.

Reply:
and all the one below that don't use source package.

build them for source. The pip generator tool does use source.

Reply:
@bbhtt , I'll rework the python-dependencies manifest to make it arch independent.

@hfiguiere , flatpak-pip-generator seems to be pulling in a lot of prebuilt wheels instead of tar files. Libraries like pyudev, pyxdg, most matplotlib dependencies are all pulled in as wheels.

Reply:
> flatpak-pip-generator seems to be pulling in a lot of prebuilt wheels instead of tar files

It only prefers `any` wheels over tars, it should not prefer arch specific wheels over tars.

Reply:
Fixed in latest commit.